src/interfaces/assistants_web/src/styles/main.css

```diff
@@ -1,12 +1,12 @@
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
-
 @import url('./fonts.css');
 @import url('./globals.css');
 @import url('./code-style.css');
 @import url('./dark-mode.css');
 
+@tailwind base;
+@tailwind components;
+@tailwind utilities;

```

Comment:
why this change? 🤔

Reply:
turbo pack requested this change

Reply:
if we make this change, the globals.css file won't work because its using the `theme`... wait a minute.. check `globals.css` we are re-importing again tailwind 🤦

Reply:
oh nice cach